rc/filetype/kotlin.kak

```diff
@@ -0,0 +1,299 @@
+    print_static_words(kotlinErrors);
+
+    print_static_words(javaBaseExceptions);
+    print_static_words(javaCompilerExceptions);
+    print_static_words(javaDatatransferExceptions);
+    print_static_words(javaIOExceptions);
+    print_static_words(javaLangExceptions);
+    print_static_words(javaLangModuleExceptions);
+    print_static_words(javaLangReflectExceptions);
+    print_static_words(javaManagementExceptions);
+    print_static_words(javaNetExceptions);
+    print_static_words(javaNioExceptions);
+    print_static_words(javaNioChannelsExceptions);
+    print_static_words(javaNioFileExceptions);
+    print_static_words(javaSecurityExceptions);
+    print_static_words(javaSecurityCertExceptions);
+    print_static_words(javaSecuritySpecExceptions);
+    print_static_words(javaSQLExceptions);
+    print_static_words(javaUtilExceptions);
+    print_static_words(javaUtilRegexExceptions);
+    print_static_words(javaXMLExceptions);
+
+    print_static_words(javaxCryptoExceptions);
+    print_static_words(javaxSecurityAuthLoginExceptions);
+    print_static_words(javaxSQLExceptions);
+
+    print_static_words(javaErrors);
+    print_static_words(javaLangErrors);
+    print_static_words(javaXMLErrors);
+    printf("\n");
+
+    add_word_highlighter(kotlinExceptions, "rgb:d26937");
+    add_word_highlighter(kotlinErrors, "rgb:d26937");
+
+    add_word_highlighter(javaBaseExceptions, "rgb:d26937");
+    add_word_highlighter(javaCompilerExceptions, "rgb:d26937");
+    add_word_highlighter(javaDatatransferExceptions, "rgb:d26937");
+    add_word_highlighter(javaIOExceptions, "rgb:d26937");
+    add_word_highlighter(javaLangExceptions, "rgb:d26937");
+    add_word_highlighter(javaLangModuleExceptions, "rgb:d26937");
+    add_word_highlighter(javaLangReflectExceptions, "rgb:d26937");
+    add_word_highlighter(javaManagementExceptions, "rgb:d26937");
+    add_word_highlighter(javaNetExceptions, "rgb:d26937");
+    add_word_highlighter(javaNioExceptions, "rgb:d26937");
+    add_word_highlighter(javaNioChannelsExceptions, "rgb:d26937");
+    add_word_highlighter(javaNioFileExceptions, "rgb:d26937");
+    add_word_highlighter(javaSecurityExceptions, "rgb:d26937");
+    add_word_highlighter(javaSecurityCertExceptions, "rgb:d26937");
+    add_word_highlighter(javaSecuritySpecExceptions, "rgb:d26937");
+    add_word_highlighter(javaSQLExceptions, "rgb:d26937");
+    add_word_highlighter(javaUtilExceptions, "rgb:d26937");
+    add_word_highlighter(javaUtilRegexExceptions, "rgb:d26937");
+    add_word_highlighter(javaXMLExceptions, "rgb:d26937");
+
+    add_word_highlighter(javaxCryptoExceptions, "rgb:d26937");
+    add_word_highlighter(javaxSecurityAuthLoginExceptions, "rgb:d26937");
+    add_word_highlighter(javaxSQLExceptions, "rgb:d26937");
+
+    add_word_highlighter(javaErrors, "rgb:d26937");
+    add_word_highlighter(javaLangErrors, "rgb:d26937");
+    add_word_highlighter(javaXMLErrors, "rgb:d26937");
+  }
+EOF
+}
+
+§
+
+# References
```

Comment:
Shouldn't these be moved to the top of the file, similarly to how other language support scripts do? A user randomly browsing language support scripts could click on the URL in the header without having to look through the file.

Reply:
Hi, that can be done for sure. However, before creating the PR I quickly browsed through other file types and I didn't find any that contained referenced, or any comments at the top. If you can point me to the style guide, or otherwise give some clues at the respective style guide, I will gladly correct this.

Reply:
Maybe it won't be necessary to move that section at all, adding a link to the official reference is probably enough, similarly to what other files do.

For example, in `asciidoc.kak`:

https://github.com/mawww/kakoune/blob/7c321e121d5910a9316cd04a6c596c1fe95b8bde/rc/filetype/asciidoc.kak#L1-L2